tutorials/01-nextjs-clerk/.env.example

```
@@ -0,0 +1,7 @@
+# Clerk Authentication
+NEXT_PUBLIC_CLERK_PUBLISHABLE_KEY=pk_test_xxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
+CLERK_SECRET_KEY=sk_test_xxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
+
+# Chipi SDK
+NEXT_PUBLIC_CHIPI_API_KEY=pk_prod_xxxxxxxxxxxxxxxxxxxx
+CHIPI_SECRET_KEY=sk_prod_xxxxxxxxxxxxxxxxxxxx
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)